docs/InstallCuda.md

```diff
@@ -0,0 +1,124 @@
+# CUDA Installation
+
+Getting CUDA and the NVIDIA driver installed correctly on your machine can be tricky.
+This guide should tell you everything you need to know (on Ubuntu, at least!).
+
+Another good resource is the [CUDA installation guide for Linux](http://docs.nvidia.com/cuda/cuda-installation-guide-linux).
+
+## GPU
+
+You will need an NVIDIA GPU if you want to use CUDA.
+**If you want to use [cuDNN](https://developer.nvidia.com/cudnn), you will need a GPU with compute capability >= 3.0.**
+To find out what the compute capability of your card is, check one of these websites:
+
+* https://developer.nvidia.com/cuda-gpus
+* https://en.wikipedia.org/wiki/CUDA#Supported_GPUs
+
+You can also use the DIGITS `device_query` tool to check for the compute `major` and `minor` versions:
+```sh
+$ digits/device_query.py
+Device #0:
+>>> CUDA attributes:
+  name                         Tesla K40c
+  totalGlobalMem               12079136768
+  clockRate                    745000
+  major                        3
+  minor                        5
+>>> NVML attributes:
+  Total memory                 11519 MB
+  Used memory                  23 MB
+  Memory utilization           0%
+  GPU utilization              0%
+  Temperature                  30 C
+```
+
+## Driver
+
+On Ubuntu, you can install a driver in two ways: with a `run` file or with a Debian package.
+
+It is recommended that you use a Debian package to install your driver, unless you have a new GPU that requires a newer driver version.
+Debian packages are simpler to install, uninstall and upgrade, while `run` file installers are useful if you need a newer driver version.
+
+To install with a `run` file, download one from the [NVIDIA Driver Downloads](http://www.nvidia.com/Download/index.aspx) website and follow the instructions.
+If you run into any problems, look at the "Additional Information" section.
+
+> **IMPORTANT**: If you use a `run` file to install your driver, don't install the `cuda` Debian package.
+More information [below](#deb-packages).
```

Comment:
Yeah we keep waffling on this throughout the documentation. Scanning through our other documentation [here](https://developer.nvidia.com/cuda-downloads) and [here](http://docs.nvidia.com/cuda/cuda-installation-guide-linux), it looks like we're not consistent there, either. Although Andy does overall seem to prefer "RPM vs. Deb" to "RPM vs. Debian".

In my mind, "deb" is an archive format. You can use some packaging tools to build "Debian deb packages" for a Debian distro, or "Ubuntu deb packages" for a Ubuntu distro.

It seems like a matter of opinion to me - I can't find anything that makes a case for "the right way" to talk about it.